upgrade-2.x.sh

```diff
@@ -302,6 +302,50 @@ function pre_update_fix_bootfiles_hook {
     mount --bind "$bootfiles_temp"  /etc/hostapp-update-hooks.d/0-bootfiles
 }
 
+function pre_update_jetson_fix {
+    log "Caching current extlinux.conf for ${SLUG} fix"
+    extlinux_path=/boot/extlinux
+    mkdir -p "/tmp${extlinux_path}"
+    cp "/mnt${extlinux_path}/extlinux.conf" "/tmp${extlinux_path}/extlinux.conf"
+    log "Stopping supervisor to prevent reboots during extlinux.conf updating"
+    stop_services
+}
+
+function parse_isolcpus {
+    path=$1
+    if grep -q "isolcpus=" "${path}" ; then
+        # shellcheck disable=SC2013
+        for val in $(awk '/isolcpus=/' "${path}"); do
+            if echo "${val}" | grep -q "isolcpus="; then
+                echo "${val}"
+            fi
+        done
+    fi
+}
+
+function post_update_jetson_fix {
+    log "Applying extlinux.conf fix for ${SLUG}"
+    # check if current config has isolcpus set in extlinux.conf
+    extlinux_file=/boot/extlinux/extlinux.conf
```

Comment:
sorry my previous comment actually probably applies to this location.

Reply:
this is a little unclear in the implementation, but it's actually targeting `/mnt${extlinux_file}` (aka `/mnt/boot/extlinux/extlinux.conf`). I'm updating the code to make this clearer, thanks for pointing this out.